### DISCUSSION TITLE
ScriptReference/Gizmos.DrawIcon

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#577] [Notes]
# ScriptReference/Gizmos.DrawIcon

https://docs.unity3d.com/ScriptReference/Gizmos.DrawIcon.html

**Comment by electronics45** [comment]
Note:  Unlike other special folders, the Gizmos folder MUST be located in the root Assets folder.  Any subfolder named "Gizmos" will not.

Note: You do not need to include the file extension.  Altering the above example, this will work too:
`Gizmos.DrawIcon (transform.position, "Light Gizmo", true);`

HACK (2020.3): If you your target icon is set as a script asset icon, DrawIcon will be able to use it regardless of which folder it's located in, but ONLY when the game object is NOT selected.
